docs/administrator-documentation/moderne-platform/how-to-guides/agent-configuration/org-service-migration.md

```diff
@@ -62,12 +58,15 @@ Make sure that the following configurations are not included in your Agent run c
 </TabItem>
 </Tabs>
 
-### 2. Configure the Agent to use `repos.csv` and `devcenter.json`
+### 2. Create the new files and put them somewhere the Agent can access
 
-1. Copy the `devcenter.json` file from your Organization Service and put it somewhere where your Agent can access.
-2. Follow [the organizational hierarchy configuration instructions](./configure-organizations-hierarchy.md) to generate a `repos.csv`. Alternatively, if your Organization Service already uses a `repos.csv`, you may copy that file directly and put it somewhere where your Agent can access.
+1. Copy the `devcenter.json` file from your Organization Service and put it somewhere where your Agent can access. This could mean putting this file on the same file system that Agent has access to – or it could mean putting it behind an unauthenticated HTTP endpoint.
```

Comment:
```suggestion
1. Copy the `devcenter.json` file from your Organization Service and put it somewhere where your Agent can access. This could mean putting this file on the same file system that Agent has access to – or it could mean putting it behind an unauthenticated HTTP/S endpoint.
```